docs/openapi/components/schemas/common/Claim.yml

```diff
@@ -0,0 +1,44 @@
+$linkedData:
+  term: Claim
+  '@id': https://w3id.org/traceability#Claim
+title: Claim
+description: A claim's subject and value.
+type: object
+properties:
+  type:
+    oneOf:
+      - type: array
+        items:
+          type: string
+          enum:
+            - Claim
+      - type: string
+        const: Claim
+  claim:
+    title: Claim
+    description: The subject of a claim.
+    type: string
+    $linkedData:
+      term: claim
+      '@id': https://schema.org/name
+  value:
+    title: Value
+    description: The value of a claim.
+    type: string
+    $linkedData:
+      term: value
+      '@id': https://schema.org/value
+  valueObject:
```

Comment:
is this the same as Value just as an object not a string?

Reply:
Yeah, Mike said values would be either strings or objects & this seemed like a reasonable approach.